index.html

```diff
@@ -0,0 +1,494 @@
+                Problems trying to resolve the conflict between <br>
+                the two major realms of Classical physics: Newtonian mechanics 
+            </div>
+            <div class="client">
+                <div class="client__card">
+                    <div class="client__card-rate">
+                        <img src="./assets/svgs/rates.svg" alt="ratings">
+                    </div>
+                    <div class="client__card-description">
+                        Slate helps you see how many 
+                        more days you need to work to 
+                        reach your financial goal.
+                    </div>
+                    <div class="client__card-profile">
+                        <img class="client__card-image" src="./assets/img/Ellipse.png" alt="client image">
+                        <div class="cleint__card-information">
+                            <div class="cleint__card-username">
+                                Regina Miles
+                            </div>
+                            <div class="cleint__card-title">
+                                Designer
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="client__card">
+                    <div class="client__card-rate">
+                        <img src="./assets/svgs/rates.svg" alt="ratings">
+                    </div>
+                    <div class="client__card-description">
+                        Slate helps you see how many 
+                        more days you need to work to 
+                        reach your financial goal.
+                    </div>
+                    <div class="client__card-profile">
+                        <img class="client__card-image" src="./assets/img/Ellipse1.png" alt="client image">
+                        <div class="cleint__card-information">
+                            <div class="cleint__card-username">
+                                Regina Miles
+                            </div>
+                            <div class="cleint__card-title">
+                                Designer
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="client__card">
+                    <div class="client__card-rate">
+                        <img src="./assets/svgs/rates.svg" alt="ratings">
+                    </div>
+                    <div class="client__card-description">
+                        Slate helps you see how many 
+                        more days you need to work to 
+                        reach your financial goal.
+                    </div>
+                    <div class="client__card-profile">
+                        <img class="client__card-image" src="./assets/img/Ellipse2.png" alt="client image">
+                        <div class="cleint__card-information">
```

Comment:
`cleint__card-information` is positioned outside the Block `(cleint)`. or it's a mistake you wanted to write `client__card-information`